### ISSUE TITLE
can't reproduce netflix content

### ISSUE THREAD

**Issue by adrianinsaval** [flathub/com.opera.Opera#127]
![image](https://github.com/user-attachments/assets/6bf1c7c2-26f9-4122-af6b-07aeaf23f601)

protected content is enabled in settings, I can only speculate that the shipped ffmpeg doesn't work properly. I use arch so I installed the AUR package for the browser and the codecs and with that netflix works with 1080p. This seems to be the only browser that plays netflix videos at 1080p but I don't care for it beyond that, it would be a ideal for me if the flathub version worked.

looking at where things are coming from for each package I see that the AUR package extracts the codecs from a snap version of chromium while this is getting it from some random github repo
https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=opera-ffmpeg-codecs-bin

perhaps you could switch to the same strategy as the aur?

**Comment by Gaeldrin** [comment]
Hello Adrian, this is actually a duplicate of #35. I mention two blocks there, the codec problem is now solved (since #98) but as far as I know there is nothing I can do about the Operation not permitted error. :(

**Comment by Phelsong** [comment]
The issue seems to be Opera requires namespace sandboxing (or no-sandbox) to correctly decrypt widevine content. The flatpak version is running in SUID mode, so it doesn't have access to widevine. 
The namespace sandbox is the default, which means its missing something, tho I'm still working out what that is.

You can see this at 
### opera://sandbox

and if you run 
```sh
flatpak run com.opera.Opera --no-sandbox
```
it can decrypt content just fine, so its not ffmpeg specifically.

**Comment by Gaeldrin** [comment]
This is interesting, thank you Phelsong for sharing this info. I tried it and can reproduce DRM content with the `--no-sandbox` flag. The reason for SUID mode is the [zypak library](https://github.com/refi64/zypak), there is a good description of how it works.

And after reading that page myself and doing some research, I think I found the solution to this problem. Looks like I had improperly configured cobalt library, the latest version works for me. It should be available on Flathub in about 3 hours.
